yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/policy/profile/IdentityProfile.java

```diff
@@ -0,0 +1,76 @@
+package com.yoti.api.client.shareurl.policy.profile;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Set of data required to represent an identity profile
+ */
+public final class IdentityProfile {
```

Comment:
In dotnet we have modelled this intentionally "loosely" using object (as opposed to having a class like IdentityProfile definied for the input). https://github.com/getyoti/yoti-dotnet-sdk/pull/354/commits/cc94506bc9496c878c8fbdfffddb464eb91b4a17 This means  Integrators can supply an anonymous object to match their needs against the API's schema; i.e. the SDK is buffered from potential changes to the json schema for the request. I am happy to approve as is, only hesitation is that the java SDK might need to be updated if these inputs change. @echarrod thoughts here please?

Reply:
Yes I believe @pgreci said this should be dynamic, as the structure might change in future iterations, and we don't want to have to update the SDKs as we add new properties to this object :+1:

Reply:
So I am not sure about the best way to achieve this dynamic nature in Java. In dotnet we simply used object for the type so Integrators can then construct with anonymus syntax as per CreateStandardIdentityProfileRequirements here https://github.com/getyoti/yoti-dotnet-sdk/pull/354/commits/cc94506bc9496c878c8fbdfffddb464eb91b4a17 @irotech what are your thoughts on what is best in Java here?